Project/Assets/ML-Agents/Editor/Cloudbuild/DisableBurstFromMenu.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if UNITY_CLOUDBUILD
+using UnityEditor;
+
+namespace MLAgents
+{
+    public class DisableBurstFromMenu
+    {
+        public static void DisableBurstCompilation()
+        {
+            EditorApplication.ExecuteMenuItem("Jobs/Burst/Enable Compilation");
```

Comment:
Can you add a comment about why we need this (and when it might be safe to remove)?